miner_exporter.py

```diff
@@ -182,12 +194,13 @@ def stats(miner: MinerJSONRPC):
     if version is not None:
         VALIDATOR_VERSION.labels(my_label).info({'version':version})
 
-    if this_validator is not None:
```

Comment:
Won't "this validator" also be in the full set? Why don't you apply the same transformations to the miner.ledger_validators(address=addr) result to turn it into a dict of dicts and then you can use one loop to output the ledger details regardless of whether it is a single (this validator) or all validators?

Reply:
I don't follow. In the new if/else starting on line 120 above, penalty_ledger contains either either the full set or just "this one". I did that so the for loop below here (starting on 198) can loop over penalty_ledger in all cases. 

The only reason I kept using `this_validator` above was because I saw that other things in the code referenced it and I was moving fast to get this up before I has to leave. `this_validator` ends up being just a copy of one entry from `penalty_ledger`. It's probably cleaner to have everything always refer to `penalty_ledger`

Reply:
kk, just fixed this up.